DayrSuryan/DSEthiop23.xml

```diff
@@ -62,99 +62,102 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                        The text contains 89 chapters or statutes (<foreign xml:lang="gez">ዓንቀጽ</foreign>). The last one concerns the order of 
-                        making changes in the statutes 
-                        (the procedure in stages, marked <foreign xml:lang="gez">ሀ</foreign>, 
+                     <note>The text is a collection of the Statutes of the Coptic Church
+                        approved on <date>2 June 1985</date> (the feast of <persName ref="PRS4306GabraMa"/>).
+                        The statutes were issued in Arabic and English; the translator, ʾabuna Maqāryos,
+                         used for his translation the Arabic version (he added English words to the Tigrinya text, to clarify some terms).
+                        The text contains 89 chapters or statutes (<foreign xml:lang="gez">ዓንቀጽ</foreign>). The last one concerns the order of
+                        making changes in the Statutes
+                        (the procedure in stages, marked <foreign xml:lang="gez">ሀ</foreign>,
                         <foreign xml:lang="gez">ለ</foreign>, <foreign xml:lang="gez">ሓ</foreign>, <foreign xml:lang="gez">መ</foreign>).  </note>
                      <incipit xml:lang="ti">
-                         በስመ፡  አብ፡  ወወልድ፡  ወመንፈስ፡  ቅዱስ፡   አሐዱ፡  አምላክ፡  አሜን ። 
-                         ዓንዲ፡  ሕጊ፡  ናይ፡  ግብጺ፡  ኦርቶዶክስ፡  ቤተ፡  ክርስቲያን፡  ሲኖዶስ። ንናይ፡  ግብጺ፡  እስክንድርያ፡  ኦርቶዶክስ፡  ቤተ፡  ክርስቲያንን፡  ናይ፡  
-                         ሓውርያ፡  ማርቆስ፡  ሃገረ፡  ስብከትን፡  ኵሉ፡  መማኀዳደሪ።.....
-                     ቀዳማይ፡  ክፍሊ፡  መእታዊ፡  (መቅድማት)
-                     ዓንቀጽ፡  (፩)
-                     ናይ፡  እስክንድርያ፡  ኦርቶዶክስ፡  ናይ፡  ግብጺ፡  ቤተ፡  ክርስቲያን፡  ሓዋርያዊት፡  ቤተ፡  ክርስቲያን፡  እያ፡  ርክባ፡  ከኣ፡  ምስተን፡  ናይ፡  ምብራቅ
-                     ፡  (oriental orthodox) ኦርቶዶክስ፡   ብእምነትን፡  <pb n="2r"/>ምሥጢራተ፡  ቤተ፡  ክርስቲያንን፡  ልማድን፡  (ecclesiastical tradition) ምሳኣ
-                     ፡  ርክብ (ኅብረት፡  ዘለወን፡  እይ     
+                         በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፡ አሜን ።
+                         ዓንዲ፡ ሕጊ፡ ናይ፡ ግብጺ፡ ኦርቶዶክስ፡ ቤተ፡ ክርስቲያን፡ ሲኖዶስ። ንናይ፡ ግብጺ፡ እስክንድርያ፡ ኦርቶዶክስ፡ ቤተ፡ ክርስቲያንን፡ ናይ፡
+                         ሓውርያ፡ ማርቆስ፡ ሃገረ፡ ስብከትን፡ ኵሉ፡ መማኀዳደሪ።...
+                     ቀዳማይ፡ ክፍሊ፡ መእታዊ፡ (መቅድማት)
+                     ዓንቀጽ፡ (፩)
+                     ናይ፡ እስክንድርያ፡ ኦርቶዶክስ፡ ናይ፡ ግብጺ፡ ቤተ፡ ክርስቲያን፡ ሓዋርያዊት፡ ቤተ፡ ክርስቲያን፡ እያ፡ ርክባ፡ ከኣ፡ ምስተን፡ ናይ፡ ምብራቅ፡
+                     (oriental orthodox) ኦርቶዶክስ፡  ብእምነትን፡ <pb n="2r"/>ምሥጢራተ፡ ቤተ፡ ክርስቲያንን፡ ልማድን፡ (ecclesiastical tradition) ምሳኣ፡
+                     ርክብ (ኅብረት፡ ዘለወን፡ እይ
                      </incipit>
                      <explicit xml:lang="ti">
                         <locus target="#24r"/>
-                        መ_እቲ፡  ዝልወጥ፡  ዓንቀጽ፡  ወይክፍሊ፡  ቅ/ፓፓ፡  ብዘይብሉ፡  ጊዜ፡  ክኸውን፡  ኣይከኣልን፡  እዩ። ተርጉዋሚኡን፡  ጻሓፊኡን፡  መቃርዮስ፡  አቡን፡  ዲበኵሉ፡  ፲፱የኲሩ 1994
+                        መ_እቲ፡  ዝልወጥ፡  ዓንቀጽ፡ ወይክፍሊ፡ ቅ/ፓፓ፡ ብዘይብሉ፡ ጊዜ፡ ክኸውን፡
```

Comment:
Is _ in the manuscript? I don't find the page in the images.

Reply:
I think it can be dropped.